tmt/utils.py

```diff
@@ -1072,6 +1069,45 @@ def quietness(self) -> bool:
 
         return self._logger.quiet
 
+    # TODO: interestingly, the option has its own default, right? So why do we
+    # need a default of our own? Because sometimes commands have not been
+    # invoked, and there's not CLI invocation to ask for the default value.
+    # Maybe se should add some kind of "default invocation"...
+    def _get_cli_flag(self, key: str, option: str, default: bool) -> bool:
+        """
+        Find the eventual value of a CLI-provided flag option.
+
+        :param key: in the tree of :py:class:`Common` instance, the
+            flag is represented by this attribute.
+        :param option: a CLI option name of the flag.
+        :param default: default value if the option has not been specified.
+        """
+
+        if self.parent:
+            parent = cast(bool, getattr(self.parent, key))
+
+            if parent:
+                return parent
+
+        invocation = self._inherited_cli_invocation
+
+        if invocation:
+            return invocation.options.get(option, default)
+
+        return default
+
+    @property
+    def is_dryrun(self) -> bool:
```

Comment:
```suggestion
    def is_dry_run(self) -> bool:
```

Let's be consistent with separating words with underscores.

Reply:
Fixed in 7aa7a2ff